internal/util/sessionutil/session_util.go

```diff
@@ -197,17 +201,27 @@ func (s *Session) MarshalJSON() ([]byte, error) {
 		Version:     verStr,
 		LeaseID:     s.leaseID,
 	})
-
 }
 
 // NewSession is a helper to build Session object.
 // ServerID, ServerName, Address, Exclusive will be assigned after Init().
-// metaRoot is a path in etcd to save session information.
-// etcdEndpoints is to init etcdCli when NewSession
-func NewSession(ctx context.Context, metaRoot string, client *clientv3.Client, opts ...SessionOption) *Session {
+// metaRoot is a path in etcd to save session information
+// client is etcdCli for NewSession
+// CLIENT AND METAROOT ARE NIL FOR NODES (only use for cmd, backup, tests)
+func NewSession(ctx context.Context, metaRoot *string, client *clientv3.Client, opts ...SessionOption) *Session {
+	var rootpath string
```

Comment:
It's probably better to (1)rename it to NewSessionWithEtcd(...)
(2) create a new function NewSession, which simply calls NewSessionWithEtcd with default metaRoot and etcd client.

So that the session creation calls don't need to pass in the two nil values.